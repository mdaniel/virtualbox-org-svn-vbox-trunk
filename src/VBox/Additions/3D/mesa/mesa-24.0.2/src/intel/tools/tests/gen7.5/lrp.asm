lrp(8)          g42<1>F         g41<4,4,1>.xF   g40<4,4,1>F     g39<4,4,1>F { align16 1Q };
lrp(16)         g4<1>F          g2.4<0,1,0>F    g2.2<0,1,0>F    g2.0<0,1,0>F { align16 1H };
lrp.sat(8)      g7<1>F          g10<4,4,1>F     g13<4,4,1>F     g16<4,4,1>F { align16 1Q };
lrp.sat(16)     g18<1>F         g20<4,4,1>F     g26<4,4,1>F     g32<4,4,1>F { align16 1H };
