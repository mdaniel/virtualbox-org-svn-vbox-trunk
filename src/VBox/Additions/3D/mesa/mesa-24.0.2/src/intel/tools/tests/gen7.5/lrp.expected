5c 01 60 00 00 00 1e 2a 00 90 02 39 50 20 c7 09
5c 01 80 00 00 00 1e 04 01 28 20 80 04 04 80 00
5c 01 60 80 00 00 1e 07 c8 a1 00 39 1a 20 07 04
5c 01 80 80 00 00 1e 12 c8 41 01 39 34 20 07 08
